steps/02/README.md

```diff
@@ -60,7 +60,7 @@ We specify the compiler options as follow:
 ```json
 {
     "compilerOptions": {
-      "target": "es2022",
+      "target": "es2023",
```

Comment:
Here I'm not sure whether we should only change `target` and keep `module` on the old ES version. This needs to be verified first what the right approach is and we need to update also our templates for that.

Reply:
After checking some documentations, IMO `target` and `module` should be on the same version for frontend applications. The `target` defines to which base version the JavaScript is transpiled to and the `module` defines the module system which can be also AMD or CommonJS besides ESxxxx.

Reply:
I checked it locally in VSCode and using "module": "es2023" leads to a "lint" error because the enum does not contain es2023. https://www.typescriptlang.org/tsconfig/#module

Reply:
AFAIK there is no change for the module system in ES2023 and I guess thats the reason why its not documented

Reply:
Took me a while to verify it, but indeed only target supports es2023 and module not - so we merge it like that.